crates/forge/tests/e2e/running.rs

```diff
@@ -847,6 +847,33 @@ fn detailed_resources_flag() {
     );
 }
 
+#[test]
+#[cfg_attr(not(feature = "scarb_since_2_10"), ignore)]
```

Comment:
I think this PR should set a CI job as well to run this

Reply:
This is done in https://github.com/foundry-rs/starknet-foundry/pull/3055 , but if you really want it here I can backport :+1:

Reply:
Not needed, I'll review it there then.